### ISSUE TITLE
QBChat disconnect does not clear currentUser and isConnecting properties

### ISSUE THREAD

**Issue by intonarumori** [QuickBlox/quickblox-ios-sdk#631]
**Environment details**
Environment: iOS10 simulator, QuickBlox 2.8.0.1 install with CocoaPods

**Did this work before?**
No idea.

**Expected behavior**
After successful `QBChat.instance().disconnect(...)` `QBChat.instance().currentUser()` should be `nil`, `isConnecting` should be `false`

**Actual behavior**
After successful disconnect `QBChat.instance().currentUser()` is still retaining the logged in user and `isConnecting` is `true`

**Steps to reproduce the behavior**

- Connect to chat with a valid username and password and wait for a successful connection.
- Disconnect from chat and observer properties

```
QBChat.instance().disconnect { [weak self] (error) in
            if let error = error {
                self?.showError(title: "Error", message: error.localizedDescription)
            } else {
                print("DISCONNECTED")
                print(QBChat.instance().currentUser()) // [QBUUser] ...
                print(QBChat.instance().isConnecting) // true
                print(QBChat.instance().isConnected) // false
            }
        }
```

**Comment by Raikerian** [comment]
Hi,

Thank you for reporting the issue, I have tested it and can confirm that it is reproducible. We will look into it to push a fix in the next update.

Right now try to build around correct value of `isConnected `.

**Comment by Raikerian** [comment]
Fixed in 2.9.
